### PR TITLE
Update react/jsx-no-bind to warning and update event handler prefix

### PR DESCRIPTION
**Pull request by andregoncalvesdev**

### CODE REVIEW

File: rules/react.js [hunk 1]
```diff
@@ -135,13 +135,13 @@ module.exports = {
         // Enforce component methods order
         'react/sort-comp': [2, {
             'order': [
-                'static-methods',
                 'lifecycle',
                 '/^(get|set)(?!(InitialState$|DefaultProps$|ChildContext$)).+$/',
                 'render',
                 '/^_render.+$/',
```
Comment (satazor) [inline rules/react.js]: I guess we should remove the _ here too.

File: rules/react.js [hunk 2]
```diff
@@ -135,13 +135,13 @@ module.exports = {
...
                 'everything-else',
                 '/^_?handle/',
```
Comment (satazor) [inline rules/react.js]: Same here.

**Review by satazor — CHANGES_REQUESTED** [review]

**Review by satazor — APPROVED** [review]

File: test/fixtures/rules/react/react.js [hunk 3]
```diff
@@ -101,19 +101,19 @@ Component.propTypes = {
 (function () {
     class MyComponent extends Component {
         render() {
-            return <a href="/foo" onClick={ this._onClick }>bleh</a>;
+            return <a href="/foo" onClick={ this.onClick }>bleh</a>;
         }
 
         _renderFoo() {
-            return <a href="/foo" onClick={ this.handleClick }>foo</a>;
+            return <a href="/foo" onClick={ this._handleClick }>foo</a>;
```
Comment (antoniocapelo) [inline test/fixtures/rules/react/react.js]: shouldn't this be `handleClick` also?

Comment (satazor) [inline test/fixtures/rules/react/react.js:108]: Nop because it's on the `// Bad` part.

File: README.md [hunk 4]
```diff
@@ -108,6 +108,10 @@ Note that by setting `root` to true, we ensure that no ancestor configuration is
 If your file exports a single class, your filename should be exactly the name of the class. For other cases, the name of the file should be the same as the default exports. If the file doesn't have a default export, the name can be in `kebab-case` (e.g.: `some-file.js`).
 
 
+## New language features
```
Comment (satazor) [inline README.md]: Could you put this inside `## Usage` with three `###`?

**Review by satazor — CHANGES_REQUESTED** [review]

**Review by satazor — APPROVED** [review]

**Pull request merged**